Playbooks/AS-PagerDuty-Integration/README.md

```diff
@@ -0,0 +1,136 @@
+# AS-PagerDuty-Integration
+
+Author: Accelerynt
+
+For any technical questions, please contact info@accelerynt.com  
+
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FPlaybooks%2FAS-PagerDuty-Integration%2Fmain%2Fazuredeploy.json" target="_blank">
```

Comment:
Incorrect link added for "Deploy to Azure". azuredeploy.json is not present in the main folder.
https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FPlaybooks%2FAS-PagerDuty-Integration%2Fmain%2Fazuredeploy.json

Deploy to Azure button is added twice to the README.md. Ensure this is present only once, preferable where steps for deployment are mentioned.

Reply:
This is resolved